app/web_framework/route_node.py

```diff
@@ -51,17 +51,35 @@ def name(self):
                     return '[{}]'.format(get_params[-1])
         return self.regex_part
 
-    def add_children(self, child_nodes):
+    def add_children(self, child_nodes: list, version: Optional[int]=None):
```

Comment:
This can be `child_nodes: List['RouteNode']` (with RouteNode in quotes).
https://www.python.org/dev/peps/pep-0484/#forward-references

Also, you can remove the docstring type hints for any annotation types you've added.